server/portal/settings/settings.py

```diff
@@ -64,6 +64,10 @@
 
 INSTALLED_APPS = [
 
+    # Django Channels
+    'channels',
+    'daphne',
+
```

Comment:
Related: https://channels.readthedocs.io/en/stable/releases/4.0.0.html#decoupling-of-the-daphne-application-server